projects/igniteui-angular/src/lib/splitter/splitter.component.ts

```diff
@@ -290,7 +290,16 @@ export class IgxSplitterComponent implements AfterContentInit {
      * This method inits panes with properties.
      */
     private initPanes() {
-        this.panes.forEach(pane => pane.owner = this);
+        this.panes.forEach(pane => {
```

Comment:
Not sure if setting this only on init is enough. If any of the related properties are changed runtime: pan `maxSize`, `minSize` or splitter `type` then this will probably not longer apply the correct constraints.

Reply:
@MayaKirova Should be good now.